src/endpoints/CoverageIntegrations/types.ts

```diff
@@ -19,6 +19,7 @@ export interface CreateRequest {
 }
 
 export interface UpdateRequest {
+    newsroom?: Newsroom['id'] | Newsroom['uuid'];
```

Comment:
```suggestion
    newsroom?: Newsroom['id'] | Newsroom['uuid'] | null;
```

Yes, can also remove it.